specification/web/resource-manager/Microsoft.Web/2016-08-01/WebApps.json

```diff
@@ -2538,11 +2831,262 @@
+        "operationId": "WebApps_GetFunctionsAdminToken",
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of web app",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "type": "string"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/functions/{functionName}": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Get function information by its ID for a specific scaled-out instance in a web site.",
+        "description": "Get function information by its ID for a specific scaled-out instance in a web site.",
+        "operationId": "WebApps_GetFunction",
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Site name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "functionName",
+            "in": "path",
+            "description": "Function name.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Function information returned",
+            "schema": {
+              "$ref": "#/definitions/FunctionEnvelope"
+            }
+          },
+          "404": {
+            "description": "Function with an name of {functionName} is not running"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "Get function information by its ID for a specific scaled-out instance in a web site.",
+        "description": "Get function information by its ID for a specific scaled-out instance in a web site.",
+        "operationId": "WebApps_CreateFunction",
+        "consumes": [
+          "application/json",
```

Comment:
It looks like all Antares "Create" APIs consume application/json, text/json, application/x-www-form-urlencoded.  I believe this is by design.